xbmc/TextureCacheJob.h

```diff
@@ -44,6 +45,7 @@ class CTextureDetails
   unsigned int width;
   unsigned int height;
   bool         updateable;
+  bool hashRevalidated;
```

Comment:
Please use `bool hashRevalidated{false};` (and line in constructor can be removed).

Reply:
I changed the whole class to default member initialization.

Reply:
Even better :smiley: